src/main/java/tp/cap5buddy/todolist/ParserUtilTodoList.java

```diff
@@ -0,0 +1,140 @@
+     * Parses description into a Description object.
+     *
+     * @param input user input.
+     * @return a Description object.
+     * @throws ParseException if description is invalid.
+     */
+    public static Description parseDescription(String input) throws ParseException {
+        if (!Description.isValidDescription(input)) {
+            throw new ParseException("Task description is invalid");
+        }
+        return new Description(input);
+    }
+
+    /**
+     * Parses type.
+     *
+     * @param input user input.
+     * @return a Type object.
+     * @throws ParseException if the type is invalid.
+     */
+    public static Type parseType(String input) throws ParseException {
+        String inputAllUpperCase = input.toUpperCase();
+        switch(inputAllUpperCase) {
+        case("ASSIGNMENT"):
+            return Type.ASSIGNMENT;
+        case("LAB"):
+            return Type.LAB;
+        case("TUTORIAL"):
+            return Type.TUTORIAL;
+        case("PROJECT"):
+            return Type.PROJECT;
+        case("STUDY"):
+            return Type.STUDY;
+        case("DAILY"):
+            return Type.DAILY;
+        default:
+            throw new ParseException("Task type is invalid");
+        }
+    }
+
+    /**
+     * Parses date into a Date object.
+     *
+     * @param input user input.
+     * @return a Date object.
+     * @throws ParseException if the date is invalid.
+     */
+    public static Date parseDate(String input) throws ParseException {
+        if (!Date.isValidDate(input)) {
+            throw new ParseException("Task date is invalid");
+        }
+        return new Date(input);
+    }
+
+    /**
+     * Parses priority.
+     *
+     * @param input user input.
+     * @return a Priority object.
+     * @throws ParseException if the priority is invalid.
+     */
+    public static Priority parsePriority(String input) throws ParseException {
+        String inputAllUpperCase = input.toUpperCase();
+        switch(inputAllUpperCase) {
+        case("HIGHEST"):
+            return Priority.HIGHEST;
+        case("HIGH"):
+            return Priority.HIGH;
+        case("NORMAL"):
+            return Priority.NORMAL;
+        case("LOW"):
+            return Priority.LOW;
+        default:
+            throw new ParseException("Task priority is invalid");
+        }
+    }
+
+    /**
+     * Parses criterion.
+     *
+     * @param input user input.
+     * @return a Criterion object.
+     * @throws ParseException if the criterion is invalid.
+     */
+    public static Criterion parseCriterion(String input) throws ParseException {
+        String inputAllUpperCase = input.toUpperCase();
+        switch(inputAllUpperCase) {
+        case("DESCRIPTION"):
+        case("DESC"):
+            return Criterion.DESCRIPTION;
+        case("DATE"):
+        case("DEADLINE"):
+            return Criterion.DATE;
+        case("PRIORITY"):
+        case("PRIO"):
+            return Criterion.PRIORITY;
```

Comment:
Is this just for short form?

Reply:
Yes, for now it is just used to make the testing easier (no need to write the whole thing every time).